db/migrate/20231124154049_migrate_validation_request_data.rb

```diff
@@ -76,7 +76,7 @@ def up
 
       vr.save(validate: false)
 
-      Document.where(additional_document_validation_request_id: request.id).each do |document|
+      Document.where(additional_document_validation_request_id: request.id).find_each do |document|
```

Comment:
This is fine here but it's an odd cop - `each` and `find_each` are not always directly replaceable as `find_each` will discard any `ORDER BY` clause to sort by id.